opennlp-tools/src/main/java/opennlp/tools/util/ext/ExtensionServiceKeys.java

```diff
@@ -17,11 +17,11 @@
 
 package opennlp.tools.util.ext;
 
-public final class ExtensionServiceKeys {
+public class ExtensionServiceKeys {
```

Comment:
@atarora Could you shortly elaborate why this class should not be `final` any more?

Reply:
Thank you for review and comments!
It looks like the 'final' was part of recent change which is honestly missed as part of merge (not intentional).
I can happily add it back in :) if required , although as per my previous comment I wasn't even too sure if this class is needed anymore.
Commit to follow!